radsat-sk/operation/subsystems/RAntenna.h

```diff
@@ -7,18 +7,28 @@
 #ifndef RANTENNA_H_
 #define RANTENNA_H_
 
+#include <stdint.h>
+
 /***************************************************************************************************
                                             DEFINITIONS
 ***************************************************************************************************/
 
-/** I2C Slave Address for Transceiver Receive Port */
-#define ANTENNA_I2C_SLAVE_ADDR (0x31)
+/** I2C Slave Address for Antenna Receive Port */
+#define ANTENNA_I2C_SLAVE_ADDR_PRIMARY (0x31)
+
+typedef struct _antennaDeploymentStatus {

```

Comment:
types follow the `underscore_separated_t` format instead of camel case

Reply:
Also, does this struct need to be public? Does anyone other than the antenna module need to know the specifics of which side is deployed? If not, then this struct typedef should go in the source file, so it's private